theme/static/js/hs_file_browser.js

```diff
@@ -945,6 +987,13 @@ function bindFileBrowserItemEvents() {
                             appKeyApp = false;
                         }
                     }
+                    if (appKeyApp || aggrApp) {
```

Comment:
do we need this `if (appKeyApp || aggrApp) {` outer check?
Don't we want to set `appKeyApp = false; aggrApp = false;` if there is `data-file-extensions`, regardless of the values of `appKeyApp` and `aggrApp`?

Reply:
Yeah we don't need the outer check. Fixed.